README_BUILD.md

```diff
@@ -125,7 +124,7 @@ Cmake Options       | Default | Description                                    |
 ```shell
 analog@precision:~/libiio$ mkdir build
 analog@precision:~/libiio/build$ cd build
-analog@precision:~/libiio/build$ cmake ../ -DCPP_BINDINGS=ON -DPYTHON_BINDINGS=ON
+analog@precision:~/libiio/build$ cmake ../ -DCPP_BINDINGS=ON
```

Comment:
I get the removal - but we should include some more instructions for those that only look at the build readme? (or at least point people to the `bindings/python/README.md` file for detailed instructions?

Reply:
That's a good idea.
I mentioned the Python bindings in the README_BUILD.md and provided a link to the Python Readme. It helps to keep python related instructions in one place.